### PR TITLE
[SERVICE-307] Upgrade to latest runtime (9.61.38.40)

### PR DESCRIPTION
**Pull request by tomrobertsOF**
- Updated service and demo runtime to API v38. 
 - Encountered a few issues due to sensitive order of operations, but those should now be fixed.

### CODE REVIEW

File: res/test/app.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
     "devtools_port": 9090,
     "runtime": {
-        "arguments": "--v=1 --enable-crash-reporting --disabled-frame-groups",
-        "version": "9.61.37.46"
+        "arguments": "--v=1 --enable-crash-reportingy",
```
Comment (ryanpetersonOF) [inline res/test/app.json]: 'reportingy'

File: src/provider/model/DesktopTabGroup.ts [hunk 2]
```diff
@@ -367,6 +367,15 @@ export class DesktopTabGroup implements DesktopEntity {
...
                 const remainingTab = existingTabs[0] === tab ? existingTabs[1] : existingTabs[0];
 
+                // Add a small delay before regrouping the remaining window. 
+                // This will push the current execution context to the end of the stack
+                // and ensure that the 'leave' event for the window has been fully 
+                // processed before regrouping. 
+                // As of v38 the ordering of the event processing changed causing 
+                // some very annoying race conditions and leaving the window ungrouped from
+                // the other snapped windows.
+                // Should be investigate further in SERVICE-xyz
```
Comment (ryanpetersonOF) [inline src/provider/model/DesktopTabGroup.ts]: Update # on SERVICE-XYZ

File: src/provider/model/DesktopWindow.ts [hunk 3]
```diff
@@ -424,21 +424,21 @@ export class DesktopWindow implements DesktopEntity {
     public async setSnapGroup(group: DesktopSnapGroup): Promise<void> {
         if (group !== this._snapGroup) {
             const wasSnapped = this._snapGroup.windows.length > 1;
-
+
```
Comment (pjbroadbent) [inline src/provider/model/DesktopWindow.ts]: Does this need an `npm run fix`? I think the linter removes this kind of whitespace.

Comment (tomrobertsOF) [reply src/provider/model/DesktopWindow.ts]: I'd done a lint, but forgot to push. Should be fixed now

File: src/provider/model/DesktopTabGroup.ts (hunk 2 above)
Comment (tomrobertsOF) [reply src/provider/model/DesktopTabGroup.ts]: Yep, need to make a story for it then will update accordingly.

Comment (tomrobertsOF) [reply src/provider/model/DesktopTabGroup.ts]: Done!

**Pull request merged**